.codecov.yml

```diff
@@ -1,7 +1,16 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
 coverage:
   status:
     project:
       default:
-        threshold: 0.1%
-    patch: false
-    changes: false
+        threshold: 1%
```

Comment:
Increasing the threshold is a workaround for me. Please take a look at my comment [here](https://github.com/openzim/mwoffliner/pull/1865#issuecomment-1657694463).

Reply:
Definitly, it's just a configuration which has proven to work for an other repository. It seems to me more appropriate, but it does not fix the problem/questioning we have with #1865